vaas/vaas/cluster/admin.py

```diff
@@ -207,6 +217,16 @@ def labels(self, obj):
 
     labels.short_description = 'Labels'
 
+    def get_domains(self, obj):
+        domains_html = ''
+        for domain in obj.domainmapping_set.all():
+            domains_html += (
+                "<a class='btn btn-success' href='/admin/cluster/domainmapping/?q=%s'>%s</a></div><br/>"
```

Comment:
maybe we can use bootstrap `label` css class? 
https://www.w3schools.com/bootstrap/bootstrap_badges_labels.asp

Reply:
Yes I change it from button to labels. Looks really nice.